src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -993,6 +993,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\IO\DisableMediaInsertionPrompt.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\MemoryFailPoint.Windows.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\Marshal.Windows.cs" Condition="'$(TargetsCoreRT)' != 'true'" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\Marshal.StringToAnsiString.cs" Condition="'$(TargetsCoreRT)' != 'true'" />
```

Comment:
Can this be part of in `Marshal.Windows.cs`, similar to Unix ?

Reply:
WideCharToMultiByte in the Unix Win32 PAL ignores `WC_NO_BEST_FIT_CHARS` and it always returns false for `usedDefaultChar`.

https://github.com/dotnet/coreclr/blob/master/src/pal/src/locale/unicode.cpp#L629